setup.py

```diff
@@ -119,6 +119,13 @@ def run(self):
         sdist.run(self)
 
 
+efilter_version = __version__
+if "bdist_msi" in sys.argv and efilter_version[1] == "!":
+  # bdist_msi does not support a setuptools epoch version so we remove it
+  # as a work around.
+  _, _, efilter_version = efilter_version.partition("!")
```

Comment:
This will install a package with a different version than the package thinks it is. Might be a bit confusing.